localization/english/religion/wc_gods_and_titles_l_english.yml

```diff
@@ -0,0 +1,421 @@
+﻿l_english:
+ ANCESTORS_NAME:0 "ancestors"
+ ANCESTORS_POSSESSIVE:0 "ancestors'"
+ 
+ DISORDER_NAME:0 "the Disorder"
+ DISORDER_POSSESSIVE:0 "the Disorder's"
+ 
+ TEMPLE_NAME:0 "temple"
+ TEMPLE_PLURAL:0 "temples"
+ 
+ EYE_NAME:0 "eye"
+ LIBRAM_NAME:0 "librams"
+ 
+ HIGH_MAGISTER_NAME:0 "High Magister"
+ MAGISTERIA_NAME:0 "Magisteria"
+ 
+ MONK_NAME:0 "monk"
+ MONK_PLURAL:0 "monks"
+ NUN_NAME:0 "nun"
+ NUN_PLURAL:0 "nuns"
+ MONASTIC_NAME:0 "monastic"
+ MONASTIC_PLURAL:0 "monastics"
+ 
+ PRIEST_NAME:0 "priest"
+ PRIEST_PLURAL:0 "priests"
+ PRIESTESS_NAME:0 "priestess"
+ PRIESTESS_PLURAL:0 "priestesses"
+ 
+ MAGISTER_NAME:0 "magister"
+ MAGISTER_PLURAL:0 "magisters"
+ 
+ GREAT_BRIGHT_BEYOND_NAME:0 "the Great Bright Beyond"
+ SHADOWLANDS_NAME:0 "shadowlands"
```

Comment:
Since it's a proper noun, Shadowlands should be capitalised.

Reply:
Changed in 1fc27eeb7917afbfb49ccee92bb12b37674b1c5f.